spec/models/ao_message_spec.cr

```diff
@@ -1,32 +1,102 @@
 require "spec"
 require "../../src/models/ao_message"
 
-include Twiliosim
+describe Twiliosim::AOMessage do
+  it "Say + Redirect" do
+    twiml = <<-XML
+      <?xml charset="utf-8"?>
+      <Response>
+        <Say>command</Say>
+        <Redirect>http://web.verboice.lvh.me:8080/VerboiceSid=123</Redirect>
```

Comment:
This and the following `web.verboice.lvh.me:8000` sholdn't be a parameter, to be able to work with staging as with the `ncd_local_gategay_simulator`?

Reply:
Indeed. I like to put realistic values in my tests, so we know what to expect, but the simulator will take whatever we give it.